### ISSUE TITLE
save and restore current crop settings

### ISSUE THREAD

**Issue by maciejrozanski** [naver/android-imagecropview#10]
Hi,

I have following scenario: I want to save crop settings for particular image, save it and then have possibility to restore it to ImageCropView.
Is it possible? I was trying with getImageViewMatrix(), I'm able to restore it using setImageMatrix, but after finger touch whole settings is reseted.

**Comment by helloyako** [comment]
@maciejrozanski 
Answer : I'm sorry. not supported.

I did not think restore function. 
That's good idea.

**Comment by brunoschalch** [comment]
Yes! please add restore function! or is there any way to implement it for now? I'm trying with zoomTo(float scale, float centerX, float centerY) but I can't figure it out.
Thank you.

**Comment by helloyako** [comment]
@maciejrozanski 
I have a good news for you.
your issues was solved by @brunoschalch.
You can check source code in develop branch and #11 

I deployed 1.1.0-SNAPSHOT to MAVEN SNAPSHOT REPO

```
repositories {
    maven { url 'https://oss.sonatype.org/content/repositories/snapshots' }
}
dependencies {
    compile 'com.naver.android.helloyako:imagecropview:1.1.0-SNAPSHOT'
}
```

**Comment by helloyako** [comment]
solved v1.1.0
